### PR TITLE
chore: update CODEOWNERS

### PR DESCRIPTION
**Pull request by moul**

### CODE REVIEW

**Review by thehowl — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 # Tendermint2 (Gno version).
 tm2/*                        @gnolang/tech-staff
+tm2/pkg                      @jaekwon @moul
```
Comment (moul) [inline .github/CODEOWNERS:9]: Reviews by @jaekwon and @moul are mandatory for merging, while others are encouraged.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -25,6 +27,8 @@ gno.land/*                   @gnolang/tech-staff
 
 # GnoVM/Gnolang.
 gnovm/*                      @gnolang/tech-staff
+gnovm/stdlibs                @jaekwon @moul
+gnovm/pkg/gnolang            @jaekwon @moul
```
Comment (moul) [inline .github/CODEOWNERS:31]: For GnoVM's core functionalities, @jaekwon and @moul's reviews are mandatory, while for `cmd/` and other packages (e.g., `gnomod`), only @gnolang/tech-staff's approval is required.

**Review by zivkovicmilos — APPROVED** [review]
We can add granular control in the coming months when we crystalize ownership of specific parts of gno (ex. gno mod - @harry-hov)

**Comment by moul** [comment]
And changed to these (in red):

CleanShot 2023-05-06 at 18 44 14@2x

**Comment by piux2** [comment]
This looks good. 

Code owners receive notifications when:

- A pull request is opened
- A draft pull request is marked as ready for review

However, code owners do not receive notifications when:

- A draft pull request is initially opened

Is there a method to alert and involve the owner when a draft pull request is opened? This can help engage the owner at an earlier stage in the conversation before significant time and resources are committed.

**Comment by moul** [comment]
> Is there a method to alert and involve the owner when a draft pull request is opened? This can help engage the owner at an earlier stage in the conversation before significant time and resources are committed.

You can prefix your PR with `WIP` and not use `draft`.

**Review by piux2 — APPROVED** [review]

**Review by albttx — APPROVED** [review]

**Pull request merged**